pt-br/tutorials/tour/singleton-objects.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: tutorial
+title: Objetos Singleton
+
+disqus: true
+
+tutorial: scala-tour
+num: 12
+
+tutorial-next: xml-processing
+tutorial-previous: pattern-matching
+language: pt-br
+---
+
+Métodos e valores que não são associados com instâncias individuais de uma [classe](classes.html) são considerados *objetos singleton*, denotados através da palavra-chave `object` ao invés de `class`.
+
+```
+package test
+
+object Blah {
+  def sum(l: List[Int]): Int = l.sum
+}
+```
+
+O método `sum` é disponível globalmente, e pode ser referenciado ou importado como `test.Blah.sum`.
+
+Objetos Singleton são um tipo de mescla e abreviação para definir uma classe de uso único, a qual não pode ser diretamente instanciada, e um membro `val` durante a definição do `object`. De fato, como `val`, objetos singleton podem ser definidos como membros de uma [trait](traits.html) ou [classe](classes.html), porém isso não é comum.
+
+Um objeto singleton pode estender classes e traits. Já uma [classe clase](case-classes.html) sem [parâmetros com tipo](generic-classes.html) por padrão irá criar um objeto singleton como o mesmo nome e com uma [`Função*`](http://www.scala-lang.org/api/current/scala/Function1.html) trait implementada.
+
+## Acompanhantes ##
+
+A maioria dos objetos singleton não estão sozinhos, mas sim associados com uma classe de mesmo nome. O “objeto singleton de mesmo nome” que uma classe case, acima mencionado, é um exemplo disso. Quando isso acontece, o objeto singleton é chamado de *objeto acompanhante* de uma classe e, a classe é chamada de *classe acompanhante* de um objeto.
+
+[Scaladoc](https://wiki.scala-lang.org/display/SW/Introduction) possui um um recurso espacial para navegar entre classes e seus acompanhantes: se o grande círculo contendo “C” ou “O” possui sua extremidade superior dobrada para baixo, você pode clicar no círculo para acessar o acompanhante.
+
+Se houver um objeto acompanhante para uma classe, ambos devem ser definidos no mesmo aquivo fonte. Por exemplo:
+
+```tut
+class IntPair(val x: Int, val y: Int)
+
+object IntPair {
+  import math.Ordering
+
+  implicit def ipord: Ordering[IntPair] =
+    Ordering.by(ip => (ip.x, ip.y))
+}
+```
+
+É comum ver instâncias de classes de tipo como [valores implicit](implicit-parameters.html), como `ipord` acima, definido no objeto acompanhante quando se segue o padrão da classe de tipo. Isso ocorre porque os membros do objeto acompanhante são incluídos por padrão na busca de valores implicitos.
```

Comment:
I think it's hard to understand `padrão classe de tipo` as `typeclasses`. Maybe use the original or put in parenthesis?